### ISSUE TITLE
Move ENS resolver creation to repository

### ISSUE THREAD

**Issue by biafra23** [gnosis/safe-android#1562]
The chains endpoint will tell us if ENS is available on a certain chainId by giving us a resolver address. We need to use this resolver address.  

Blocked by: https://github.com/gnosis/safe-config-service/issues/119

**Comment by biafra23** [comment]
Duplicate of 1557
